pyproject.toml

```diff
@@ -145,6 +140,10 @@ line-length = 88
 # Don't automatically concatenate strings -- sometimes we forget a comma!
 unfixable = ["ISC"]
 
+[tool.ruff.format]
+quote-style = "double"
+exclude = ["migrations/versions/*"]
```

Comment:
I don't think we have alembic migrations to ignore in this repo do we?

Reply:
Yeah no alembic in this repo

Reply:
Should we remove configurations that aren't relevant to the repo they're in?

Reply:
Oh yeah, whoops, removed